Hermes/appscale/hermes/stats/unified_service_names.py

```diff
@@ -0,0 +1,165 @@
+
+  def recognize_haproxy_proxy(self, proxy_name):
+    """ Checks whether haproxy proxy corresponds to this service """
+    return self.haproxy_proxy_matcher.match(proxy_name) is not None
+
+  def get_application_id_by_monit_name(self, monit_name):
+    """ Parses monit_name and returns application ID if it was found """
+    match = self.monit_matcher.match(monit_name)
+    if not match:
+      return None
+    try:
+      return match.group('app') if match else None
+    except IndexError:
+      return None
+
+  def get_port_by_monit_name(self, monit_name):
+    """ Parses monit_name and returns port if it was found """
+    match = self.monit_matcher.match(monit_name)
+    try:
+      port_group = match.group('port') if match else None
+      return int(port_group) if port_group else None
+    except IndexError:
+      return None
+
+  def get_application_id_by_pxname(self, pxname):
+    """ Parses haproxy proxy and returns application ID if it was found """
+    match = self.haproxy_proxy_matcher.match(pxname)
+    if not match:
+      return None
+    try:
+      return match.group('app') if match else None
+    except IndexError:
+      return None
+
+  def get_ip_port_by_svname(self, svname):
+    """ Parses haproxy proxy and returns private IP and port if it was found """
+    match = self.haproxy_server_matcher.match(svname)
+    if not match:
+      return None, None
+    try:
+      ip = match.group('ip')
+    except IndexError:
+      ip = None
+    try:
+      port_group = match.group('port')
+      port = int(port_group) if port_group else None
+    except IndexError:
+      port = None
+    return ip, port
+
+
+class ServicesEnum(object):
+  # Known by both (Monit and HAProxy)
+  UASERVER = Service(
+    name='uaserver', monit_matcher='^uaserver$',
+    haproxy_proxy_matcher=r'^UserAppServer$',
+    haproxy_server_matcher=r'^UserAppServer-(?P<ip>[\d.]+):(?P<port>\d+)$'
+  )
+  TASKQUEUE = Service(
+    name='taskqueue',
+    monit_matcher=r'^taskqueue-(?P<port>\d+)$',
+    haproxy_proxy_matcher='^TaskQueue$',
+    haproxy_server_matcher=r'^TaskQueue-(?P<ip>[\d.]+):(?P<port>\d+)$'
+  )
+  DATASTORE = Service(
+    name='datastore', monit_matcher=r'^datastore_server-(?P<port>\d+)$',
+    haproxy_proxy_matcher='^appscale-datastore_server$',
+    haproxy_server_matcher=r'^appscale-datastore_server-(?P<ip>[\d.]+):(?P<port>\d+)$'
+  )
+  BLOBSTORE = Service(
+    name='blobstore', monit_matcher='^blobstore$',
+    haproxy_proxy_matcher='^as_blob_server$',
+    haproxy_server_matcher=r'^as_blob_server-(?P<ip>[\d.]+):(?P<port>\d+)$'
+  )
+  APPLICATION = Service(
+    name='application',
+    monit_matcher=r'^app___(?P<app>[\w_-]+)-(?P<port>\d+)$',
+    haproxy_proxy_matcher=r'^gae_(?P<app>[\w_-]+)$',
+    haproxy_server_matcher=r'^gae_(?P<app>[\w_-]+)-(?P<ip>[\d.]+):(?P<port>\d+)$'
+  )
+
+  # Known only on Monit side
+  ZOOKEEPER = Service(name='zookeeper', monit_matcher='^zookeeper$')
```

Comment:
Are these Monit service names already existing constants somewhere that we can make use of. If not, do you see it useful to make them constants so that we can have them in one single place?